lib/seccomp/CSystemCallFilter_Windows.cc

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+#include <core/CWindowsError.h>
+
+#include <WinBase.h>
```

Comment:
Please use `core/WindowsSafe.h`.  If you look at the Microsoft documentation it says to include `Windows.h` rather than `WinBase.h`, so directly including `WinBase.h` may cause some transitive requirements to be missing in a different Windows SDK version.  `core/WindowsSafe.h` is a wrapper around `Windows.h` that stops it messing up some of our code.